sync/src/main/kotlin/gropius/sync/AbstractSync.kt

```diff
@@ -530,48 +653,51 @@ abstract class AbstractSync(
      * @param issueInfo Issue to sync
      * @param label Label to sync
      */
-    private suspend fun syncOutgoingSingleLabel(
+    private suspend
```

Comment:
?

Reply:
accidentally hit the big key. i am used to formatter simply frying over such shit. eaa02aceb6955c281f4129b263ed1f2d6d63ea83